NewsApp/NewsApp/Controller/ProfileController.swift

```diff
@@ -0,0 +1,61 @@
+//
+//  ProfileController.swift
+//  NewsApp
+//
+//  Created by Josselin DUBOC on 26/03/2020.
+//  Copyright © 2020 Josselin DUBOC. All rights reserved.
+//
+
+import Foundation
+import UIKit
+
+class ProfileController: UITableViewController {
+    @IBOutlet private weak var button: UIButton!
+    @IBOutlet private weak var picker: UIPickerView!
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        if let country = UserDefaults().string(forKey: "Country") {
+            button.setTitle(country, for: .normal)
+        }
+        picker.isHidden = true
+    }
+
+    @IBAction func onClick(_ sender: UIButton, forEvent event: UIEvent) {
```

Comment:
en term de naming, il serait peut-être plus clair de préciser le bouton qui est click ici
par exemple categoryButtonClicked ou onCategoryClicked

Les arguements sender et event ne sont pas utilisé et pas obligatoire, donc tu pourrais juste les omettre et mettre categoryButtonClicked() par exemple

Reply:
fix in "fix PR"